Makefile.riscv64

```diff
@@ -2,8 +2,8 @@ CC=$(HOME)/cheri/output/sdk/bin/clang
 CFORMAT=$(HOME)/cheri/output/sdk/bin/clang-format
 CXX=$(HOME)/cheri/output/sdk/bin/clang++
 CFLAGS=-fuse-ld=lld --config cheribsd-riscv64-purecap.cfg
-SSHPORT=10021
-export 
+SSHPORT=10013
```

Comment:
Is there any reason to change the port number?

Reply:
You are absolutely right. That's my port. I will fix it.

Reply:
Fixed in 56f2d0a.